game/Submods/NSFW Submod/nsfw_brb.rpy

```diff
@@ -0,0 +1,106 @@
+# BRB - Going to masturbate
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_brb_nsfw_masturbate",
+            prompt="I'm going to go masturbate",
+            category=['be right back'],
+            conditional="mas_canShowRisque()",
```

Comment:
The conditional is missing an action below it, so it will just always unlock / pool no mattter what your affection is currently.